src/sparseml/modifiers/quantization/gptq/pytorch.py

```diff
@@ -156,7 +156,7 @@ def apply_compression(
                 layer_compressor.pre_compress()
                 _LOGGER.info(f"Calibrating {layer_compressor.name}...")
                 run_calibration_forward(self.model, dataloader, mask_padding=True)
-            layer_compressor.compress()
+            layer_compressor.compress(self.actorder)
```

Comment:
is this flag also relevant in the context of SparseGPT or only GPTQ?

Reply:
Probably just GPTQ - activation reordering is for quantization flow